aggregations-and-analytics/10/10_windowing_tvf.md

```diff
@@ -0,0 +1,64 @@
+# 10 Windowing table-valued functions
+
+![Twitter Badge](https://img.shields.io/badge/Flink%20Version-1.13%2B-lightgrey)
+
+> :bulb: This example will show how to apply computations on data in a temporal window using Windows table-valued functions `TUMBLE`, `HOP` and `CUMULATE`.
+
+The source table (`server_logs`) is backed by the [`faker` connector](https://flink-packages.org/packages/flink-faker), which continuously generates rows in memory based on Java Faker expressions.
+
+#### What are Windowing Table-valued functions (TVFs)?
+
+Windows split the stream into “buckets” of finite size, over which we can apply computations.
+[Windowing TVFs](https://ci.apache.org/projects/flink/flink-docs-stable/docs/dev/table/sql/queries/window-tvf/) are Flink defined Polymorphic Table Functions (abbreviated PTF). PTF are a powerful feature to change the shape of a table.
+Previous recipes ([1](../01/01_group_by_window.md)) ([2](../02/02_watermarks.md)) ([3](../03/03_group_by_session_window.md)) are all so-called Grouped Window Functions which are considered legacy. 
+
+#### Using Windowing Table-valued functions (TVFs)
+
+There are three examples to count the number of "Forbidden" (403) requests using different Windows TVFs.
+
+* Using `TUMBLE` to provide the amount of 403 errors every 5 seconds. This means that all 403 errors that occur within 5 seconds of the last seen request will be merged in the same window; and any request that occurs outside of this gap will trigger the creation of a new session window. 
+* Using `HOP` to provide every 5 seconds the count of 403 errors over the last 10 seconds. This means that all 403 errors that occurred within 5 seconds of the last seen request will be merged in the same window, which will be displayed every 10 seconds. 
```

Comment:
Doesn't the second sentence contradict the first sentence here?

First Sentence: Every 5 Seconds, Window Size 10 Seconds
Second Sentence: Every 10 Seconds, Window Size 5 Seconds